scanners/trustymail.py

```diff
@@ -42,10 +38,10 @@ def init_domain(domain, environment, options):
             int(port)
             for port in options.get('smtp_ports', default_smtp_ports).split(',')
         }
-        dns_hostnames = options.get('dns', default_dns).split(',')
-        # Note that we _do not_ use the system configuration in
-        # /etc/resolv.conf.
-        resolver = dns.resolver.Resolver(configure=False)
+        dns_hostnames = options.get('dns')
```

Comment:
Why do we duplicate this `options.get(...)` and `...split(',')` when the same happens below at https://github.com/18F/domain-scan/pull/296/files#diff-37d61b45847fea9081213c0878237857R131 ?

Reply:
`init_domain()` runs on the local machine, while `scan()` potentially runs in a different Python interpreter in AWS Lambda.

I will pull the common functionality out into its own function.

Reply:
There was actually an existing function I could use for this.  See b840473.